packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -1,55 +1,114 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ * 
+ * @prop --calcite-input-date-picker-border-color: Specifies the component's input border color.
+ * @prop --calcite-input-date-picker-divider-color: Specifies the component's divider color between two inputs when in range mode.
+ *
+ * @prop --calcite-input-date-picker-shadow: Specifies the component's input shadow.
+ * @prop --calcite-input-date-picker-calendar-shadow: the component's calendar shadow.
+ * 
+ * @prop --calcite-input-date-picker-icon-color: Specifies the component's in-field input icon color.
+ * @prop --calcite-input-date-picker-icon-color-hover: Specifies the component's in-field input icon color on hover.
+ * @prop --calcite-input-date-picker-actions-icon-color: Specifies the component's actions icon color.
+ * @prop --calcite-input-date-picker-actions-icon-color-hover: Specifies the component's actions icon color on hover.
+ * @prop --calcite-input-date-picker-calendar-icon-color: Specifies the component's calendar icon color.
+ * @prop --calcite-input-date-picker-calendar-icon-color-hover: Specifies the component's calendar icon color.
+ * 
+ * @prop --calcite-input-date-picker-background-color: Specifies the component's input background color.
+ * @prop --calcite-input-date-picker-calendar-actions-background-color: Specifies the background color of actions of the calendar.
+ * @prop --calcite-input-date-picker-calendar-actions-background-color-hover: Specifies the background color of actions of the calendar when hovered.
+ * @prop --calcite-input-date-picker-calendar-actions-background-color-press: Specifies the background color of actions of the calendar when pressed.
+ * @prop --calcite-input-date-picker-calendar-selected-background-color: Specifies the background color of selected day and day range of the calendar.
+ * @prop --calcite-input-calendar-day-background-color: Specifies the background color of day of the calendar.
+ * @prop --calcite-input-date-picker-calendar-day-background-color-hover: Specifies the background color of day of the calendar when hovered.
+ * @prop --calcite-input-date-picker-calendar-day-outside-range-background-color-hover: Specifies the background color of day's outside current range of the component's date-picker when hovered.
+ * 
+ * @prop --calcite-input-date-picker-corner-radius: Specifies the component's input corner radius.
+ * @prop --calcite-input-date-picker-calendar-corner-radius: Specifies the component's calendar border radius.
```

Comment:
Slight difference in prop descriptions here - "input corner radius" vs "calendar border radius"